### PR TITLE
validator update from 9.0.0 to 13.7.0

### PR DESCRIPTION
**Pull request by Reni88**
Remediates CVE-2021-3765

### CODE REVIEW

**Comment by tejashah88** [comment]
@mreinstein I've noticed that I don't see the CI action workflow executing when new PRs are made. Is this intentional?

**Comment by mreinstein** [comment]
Currently it only runs on pushes to the`master` branch: https://github.com/alexa-js/alexa-verifier-middleware/blob/417f4ebfd3f1513f5c9b2b2e81dc637ccc735361/.github/workflows/main.yml#L7-L9

**Comment by mreinstein** [comment]
I've updated the workflow to run on PR activity now.

**Comment by Reni88** [comment]
Hello. Hoping someone is already reviewing this PR.

**Comment by dblock** [comment]
@Reni88 can you please rebase so we can see CI/CD kick after @mreinstein's fix of the workflow file?

File: package-lock.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "alexa-verifier-middleware",
-  "version": "2.0.1",
+  "version": "2.0.2",
```
Comment (dblock) [inline package-lock.json]: Looks like we had released 2.0.2 before, so this should be 2.0.3, the next developer iteration.

Comment (Reni88) [reply package-lock.json]: Had changes this to 2.0.3

Comment (Reni88) [reply package-lock.json]: Should I need to update package.json and changeLog too?

Comment (dblock) [reply package-lock.json]: Probably.

Comment (Reni88) [reply package-lock.json]: Done.

**Comment by tejashah88** [comment]
@dblock LGTM! You may merge when ready.

**Comment by dblock** [comment]
@tejashah88 you too :)

**Comment by Reni88** [comment]
As this is merged, @tejashah88 or @dblock , when can we expect it'll be updated in npmjs?

**Pull request merged**